### PR TITLE
Implementation of output intermediate

### PR DESCRIPTION
**Pull request by zrock-dev**
# Task 138 | Implementation of output intermediate

https://tree.taiga.io/project/joseluis-teran-coffeetime/task/138

#### What did I do it?
To enable bridge send data to screen
#### How did I do it?
Frame serialization implementation
	- The frames are serialized using flatbuffer library.
	- CoffeeTime is machine generated

implemented a socket client
	- Transmitter takes a message and proceeds to send it through the
	  SocketClient as an array of bytes

Added [flatbuffer](https://flatbuffers.dev) dependency

Generated files will be stored at src/main/java/CoffeeTime.
Updated spotless to ignore CoffeeTime

#### Why did I do it?
To enable game graphics to be rendered in Screen.

## Type of Change

Put an `x` in all the boxes that apply:

- [x] ✨ New Feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug Fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking Change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code Refactor
- [ ] ✅ Build Configuration Change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

## Pull Request Checklist

- [x] My commit messages are detailed.
- [x] My code follows the code style of this project.
- [x] No existing features have been broken without good reason.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests pass.

### CODE REVIEW

**Comment by zrock-dev** [comment]
The CoffeeTime folder has been generating using [flatcc](https://github.com/dvidelabs/flatcc), and the schema is written in:

https://github.com/Pending-Name-21/miscellany/pull/15

**Comment by zrock-dev** [comment]
This implementation doesn't have unit tests because it is required to use `Sprite` and `Sound` classes, but the factories are returning empty objects.

I've created a PR with a refactor to solve this issue.
https://github.com/Pending-Name-21/arquitecture/pull/19

**Review by Misax148 — APPROVED** [review]
Good job

**Review by sebastian-barra-jalafund — APPROVED** [review]
No observations, just add a socket test

**Review by sebastian-barra-jalafund — APPROVED** [review]
Test were added, good job

**Comment by zrock-dev** [comment]
I forgot to update RenderManager to use the Transmitter.

**Comment by zrock-dev** [comment]
I have updated this branch with develop. And updated RenderManager to use Transmitter

**Review by sebastian-barra-jalafund — APPROVED** [review]

File: bridge/src/main/java/com/bridge/ipc/SocketClient.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import java.nio.channels.SocketChannel;
+import java.nio.file.Path;
+
+/**
+ * This class provides a client-side connection for inter-process communication (IPC)
+ * using Unix domain sockets.
+ */
+public class SocketClient {
+    public static final Path NAMESPACE =
+            Path.of(System.getProperty("java.io.tmpdir"), "events-socket.sock");
```
Comment (Misax148) [inline bridge/src/main/java/com/bridge/ipc/SocketClient.java]: The path of the server-socket of Screen is `/tmp/socket_console` and should be the same.
This path I think is of input-intermediate socket

Comment (zrock-dev) [reply bridge/src/main/java/com/bridge/ipc/SocketClient.java]: I've updated it to the suggested path

**Review by Misax148 — APPROVED** [review]

**Pull request merged**